index.html

```diff
@@ -61,6 +61,164 @@ <h2 class="lets-connect">LET’S CONNECT</h2>
+          <li><a href=""><img src="./images/javascript.svg" alt="javascript-tag"></a></li>
+        </ul>
+        <button type="button" class="btn-card">See Project</button>
+      </section>
+    </section>
+
+    <section class="work-section">
+      <div class="image1">
+        <img src="images/snapshot3.svg" alt="snapshot3-image">
+      </div>
+      <section class="left-block">
+        <h2 class="title"> Tonic </h2>
+        <ul class="menu2">
+          <li><a class="canopy" href="#">CANOPY</a> </li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li><a class="back-end" href="#">Back End Dev</a></li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li> <a class="year" href="#">Year</a></li>
+        </ul>
+        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups
+          required.</p>
+        <ul class="html-tags">
+          <li><a href=""><img src="./images/html.svg" alt="html-tag"></a></li>
+          <li><a href=""><img src="./images/css.svg" alt="css-tag"></a></li>
+          <li><a href=""><img src="./images/javascript.svg" alt="javascript-tag"></a></li>
+        </ul>
+        <button type="button" class="btn-card">See Project</button>
+      </section>
+    </section>
+
+    <section class="work-section">
+      <div class="image1">
+        <img src="images//snapshot4.svg" alt="stories-image">
+      </div>
+      <section class="left-block">
+        <h2 class="title"> Multi-Post
+          Stories </h2>
+        <ul class="menu2">
+          <li><a class="canopy" href="#">CANOPY</a> </li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li><a class="back-end" href="#">Back End Dev</a></li>
+          <li><img src="./images/Counter.svg" alt="counter"></li>
+          <li> <a class="year" href="#">Year</a></li>
+        </ul>
+        <p class="paragraph2">A daily selection of privately personalized reads; no accounts or sign-ups
+          required.</p>
+        <ul class="html-tags">
+          <li><a href=""><img src="./images/html.svg" alt="html-tag"></a></li>
+          <li><a href=""><img src="./images/css.svg" alt="css-tag"></a></li>
+          <li><a href=""><img src="./images/javascript.svg" alt="javascript-tag"></a></li>
+        </ul>
+        <button type="button" class="btn-card">See Project</button>
+      </section>
+    </section>
+    
+    <Section class="about-myself">
+      <div class="header-myself">
+          <h2 class="heading-about">About <br> Myself</h2>
+      </div>
+
+      <div class="paragraph3">
+          <p class="p3">Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If you like what you see and
+              have a
+              project you need coded, don’t hestiate to contact me.
+          </p>
+          <h2 class="lets-connect">LET’S CONNECT</h2>
```

Comment:
### Your About me Section
![contact (copy)](https://user-images.githubusercontent.com/93039359/185511043-35b07b78-c8ff-4ea3-bac3-78ebd7fb52df.png)

### Figma About me Section
![FIGMA2](https://user-images.githubusercontent.com/93039359/185511062-40df4b4f-947b-4cbc-874c-6f246c723528.png)


- [ ] You can see that red boxes shows that desiging is not very similar to the Figma tempate.
- [ ] You can see that purple boxes shows the right position for the contents :thinking:

**Please make these changes to make your project look like Figma template means professional :+1:**